From cf213833ea10fd8e04b2184b996661309474478b Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Sat, 24 Feb 2018 20:43:28 -0500
Subject: [PATCH 111/249] wiki: jfltevzw: Update Custom TWRP

* This is LOKI'd, as well as includes the /misc/ fstab patch to allow OTA.
* This is temporary until TWRP rekick's off our official build to include the /misc/ patch, which was missed.

Change-Id: I85fa82e32c5d6ecf214d8c0a8605ba1c0e836428
---
 _data/devices/jfltevzw.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_data/devices/jfltevzw.yml b/_data/devices/jfltevzw.yml
index 5344a2b..3afbe0a 100644
--- a/_data/devices/jfltevzw.yml
+++ b/_data/devices/jfltevzw.yml
@@ -11,7 +11,7 @@ cpu: Krait 300
 cpu_cores: 4
 cpu_freq: 1.9GHz
 current_branch: 14.1
-custom_twrp_link: https://www.androidfilehost.com/?fid=889764386195900221
+custom_twrp_link: https://www.androidfilehost.com/?fid=962157660013071010
 depth: 7.9mm (0.31 in)
 download_boot: With the device powered off, hold <kbd>Home</kbd> + <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 320
-- 
2.17.1

